src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -768,7 +768,16 @@ def get_pending_clones(self, snapname):
 
         for track_id in pending_track_id_list:
             try:
-                link_path = self.fs.readlink(os.path.join(index_path, track_id), 4096)
+                path = os.path.join(index_path, track_id)
+                link_path = self.fs.readlink(path, 4096)
+                try:
+                    self.fs.stat(link_path)
+                except cephfs.Error as e:
+                    if e.args[0] == errno.ENOENT:
+                        self.fs.unlink(path)
```

Comment:
any cephfs.Error exception needs to be catched

Reply:
Yup, i will add